server/controllers/changeLocalName/details.ts

```diff
@@ -40,6 +40,9 @@ export default class Details extends FormInitialStep {
 
       const validationErrors: any = {}
 
+      if (!sanitizedLocalName) {
+        return callback({ ...errors, ...validationErrors })
```

Comment:
Previously, an empty local name would still have sent a request to locations API.